README.md

```diff
@@ -7,7 +7,7 @@ Highly customizable open source theme for Hugo based static websites
 **NOTE: This is a beta release, we do not consider it completely production ready yet. Use at your own risk.**
 
 # Syna Theme
-Base your next personal, project or commercial website on Syna.  
+Base your next personal, project or commercial website on Syna.
```

Comment:
The trailing whitespace is intentional due to Gitlab and other markdown implementations using that as a new line.

Reply:
oh, Alright I will fix that.